### ISSUE TITLE
Science graphs don't work well on mobile devices

### ISSUE THREAD

**Issue by MikeMcC399** [corona-warn-app/cwa-website#1436]
## Where to find the issue

https://www.coronawarn.app/de/science/2021-06-15-science-blog-1/

## Describe the issue

The dynamic graphs in https://www.coronawarn.app/de/science/2021-06-15-science-blog-1/ such as "Abb. 1: Über die CWA registrierte Tests und deren Ergebnisse." do not work well on mobile devices.

Under Google Chrome on Android 11 it is difficult to scroll across to see the whole graph.  Only by dragging above or below the graph allows scrolling the graph. If you try to scroll by dragging the graph itself, it zooms in or does other things which I haven't understood.

## Suggested change

Consider displaying static scrollable graphs on mobile devices.

---
Internal Tracking ID: [EXPOSUREAPP-8297](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-8297)

**Comment by dsarkar** [comment]
@MikeMcC399 FYI PR https://github.com/corona-warn-app/cwa-website/pull/1934

**Comment by MikeMcC399** [comment]
Replace interactive figs on mobile for images #1934 was merged and I tried it on different Samsung Galaxy models. In portrait mode the message "This page is optimized for mobile devices ..." appears and the graphs are static. That works well.

In Google Chrome viewing https://www.coronawarn.app/en/science/2021-08-02-science-blog-3/ in landscape mode with:
- Galaxy A5 (2017) - the page is optimized with static graphs
- Galaxy A50 - the page is not optimized and the graphs are interactive

At this time, I think that the changes are a big improvement compared to earlier, so probably we should stop trying to further optimize the display.

Unless there are some objections I will close this issue this week. Thanks very much to @brifemu for making these changes!
